docs/guides/admin/docs/configuration/index.md

```diff
@@ -12,7 +12,7 @@ General Configuration
  - [Database Configuration](database.md)
  - [Encoding Profile Configuration](encoding.md)
  - [Load Configuration](load.md)
- - [Logging and Privacy Configuration](logging.and.privacy.md)
+ - [Logging and Privacy Configuration](user-statistics.and.privacy.md)
```

Comment:
Here, you probably want to use “User Statistics and Privacy” as well

Reply:
you suggest to rename the file to `user.statistics.and.privacy.md`?

Reply:
forget my last comment...